client/src/components/sources/index.tsx

```diff
@@ -152,10 +152,10 @@ const SourcesView = (): React.ReactElement => {
             setSourceToDelete(undefined);
           }}
         >
-          <p>
-            Etes-vous sûr de vouloir supprimer la source{' '}
-            <b>{sourceToDelete ? sourceToDelete.name : ''}</b>? Cette action
-            n'est pas réversible.
+          <p style={{"color": "#A82A2A"}}>
```

Comment:
What's the clean way to set a color using the standard buleprint js color set?

Reply:
you should add a class or an id to your HTML element and add styling in the style.css file associated, where you can use "colors variables" directly from blueprint (there is already an example in `.../Comments/style.css`)

Reply:
It was in front of my eyes haha
Ok it's done :)